_commands/basics/grep.md

```diff
@@ -0,0 +1,62 @@
+---
+---
+
+grep
+-------
+
+`grep` is used to search files for the occurence os string of characters matching a special pattern (sometimes called a regex or regular expression).
```

Comment:
Typo? Also note that grep doesn't support full regex – that's egrep. Would make note somewhere in here.